i18n/en.json

```diff
@@ -4,6 +4,8 @@
 	},
 	"ext-whowrotethat-activation-link": "Who Wrote That?",
 	"ext-whowrotethat-activation-link-tooltip": "Activate Who Wrote That?",
+	"ext-whowrotethat-deactivation-link": "Close Who Wrote That?",
```

Comment:
OK, totally not a merge stopper, but we should flag this for Ilana and @prtksxna  ... this looks so weird as a question ...

Reply:
Yes, and 'Close Who Wrote That?' with a tooltip of 'Deactivate Who Wrote That?' seems redundant. Maybe we don't have to change the link text at all?

Reply:
I don't think there's need for a tooltip for a text-based link. The tooltip is needed when we have an icon or button that the screen reader needs help parsing...

Reply:
Yeah that makes sense. Do you mean we should remove the existing tooltip as well? Although I guess that might be sorted out in [T231612](https://phabricator.wikimedia.org/T231612). I'll remove the new tooltip here.